src/accountScreen/accountScreen.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './accountScreen.css'
+import SearchBar from 'material-ui-search-bar'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+
+
+const ChatCard = function(props) {
+  if(props.selected){
+    return (
+      <Card style={{
+        backgroundColor:'#cacaca'}}>
+        <CardHeader
+          title={props.title}
+          avatar={props.avatar}
+        />
+        <CardText>
+          {props.text}
+        </CardText>
+      </Card>        
+    );
+  }
+
+  else{
+    return (
+      <Card>
+        <CardHeader
+          title={props.title}
+          avatar={props.avatar}/>
+        <CardText>
+          {props.text}
+        </CardText>
+      </Card>        
+    );
+  }
+  
+};
+
+class AccountScreen extends Component {
+  
+
+
+  render() {
+    return (
+      <MuiThemeProvider>       
+        <div className='rectangle-4'>
+          <div className='searchContainer'>
+           <SearchBar hintText='Search Chats'/>

```

Comment:
fix indentation. perhaps install some sort of linter to check

Reply:
Indentation fixed